#List of available guidance frequencies
# guidance_frequency_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 MSG_GUID_0_RATE   = 0 #  Message is not being outputted
	uint8 MSG_GUID_100_RATE = 1
	uint8 MSG_GUID_200_RATE = 2
	uint8 MSG_GUID_300_RATE = 3
	uint8 MSG_GUID_400_RATE = 4
	uint8 MSG_GUID_500_RATE = 5
	uint8 MSG_GUID_600_RATE = 6
	uint8 MSG_GUID_250_RATE = 7
